### ISSUE TITLE
Scaling learning rate for multigpu training

### ISSUE THREAD

**Issue by sharathadavanne** [NVIDIA/waveglow#183]
Has anyone tried scaling the learning rate with the same factor as the minibatch size for multi GPU training? Similar to - https://arxiv.org/pdf/1706.02677.pdf

Frankly, I have only seen papers showing successful results with scaled learning rates for CNN based models for vision.

**Comment by CookiePPP** [comment]
My Config
```
"data_config": {
        "training_files": "map_0.txt",
        "segment_length": 9600,
        "sampling_rate": 48000,
        "filter_length": 2400,
        "hop_length": 600,
        "win_length": 2400,
        "mel_fmin": 50.0,
        "mel_fmax": 16000.0
    },
    "dist_config": {
        "dist_backend": "nccl",
        "dist_url": "tcp://127.0.0.1:54321"
    },
    "waveglow_config": {
        "n_mel_channels": 160,
        "n_flows": 12,
        "n_group": 8,
        "n_early_every": 4,
        "n_early_size": 2,
        "WN_config": {
            "n_layers": 9,
            "n_channels": 256,
            "kernel_size": 3
        }
    }
```

![paste](https://user-images.githubusercontent.com/42448678/74260517-87f48600-4cf1-11ea-8369-3a295aed3871.png)
You may run at a higher learning rate (make sure you have some warmup). These were trained batch_size 10 on 3x 2080 Ti's (therefore minibatch of 30).
Dark Blue was stopped early so I can't provide insight past the beginning of training.
